ts/Core/Animation/Fx.ts

```diff
@@ -533,6 +541,19 @@ interface Fx extends FxLike {
     // Nothing here yet
 }
 
+/* *
+ *
+ *  Compatibility
+ *
+ * */
+
 H.Fx = Fx;
+(H as any).timers = Fx.timers;
```

Comment:
I don't think we need to keep this on the namespace. If the tests pass without it I'd say we remove it.

Reply:
test-utilities requires an update then. (highcharts and highcharts-utils repositories)

Reply:
Okay. We'll leave it for now then.